package.json

```diff
@@ -18,7 +18,7 @@
   },
   "version": "0.0.1",
   "main": "build/index.js",
-  "repository": "git@github.com:goa-codebet/nimble.git",
+  "repository": "https://github.com/goa-codebet/nimble-utils",
```

Comment:
I think we should use ssh here and not https.

Reply:
Totally agree, missed that change.